.cirrus.yml

```
@@ -124,40 +124,40 @@ task:
+  name: '[no depends, sanitizers: fuzzer,address,undefined,integer] [noble]'
   << : *GLOBAL_TASK_TEMPLATE
   container:
-    image: ubuntu:jammy
+    image: ubuntu:noble
     cpu: 4  # Increase CPU and memory to avoid timeout
     memory: 16G
   env:
     MAKEJOBS: "-j8"
     FILE_ENV: "./ci/test/00_setup_env_native_fuzz.sh"
 
 task:
```

YAMLlint (1.35.1)

[error] duplication of key "task" in mapping

(key-duplicates)